build/helper/metadata_add_all.py

```diff
@@ -259,6 +269,19 @@ def add_all_enum_metadata(enums, config):
         for m in dir(config['modules']['metadata.enums_addon']):
             if m.startswith('enums_'):
                 merge_dicts(enums, config['modules']['metadata.enums_addon'].__getattribute__(m))
+            # We need to explicitly copy new entries
```

Comment:
This function is suspiciously similar to add_all_function_metadata()
I think we could probably have a single merging function, even if it means we rename the dictionaries in the metadata (i.e. this looks for things that start with "enums_"... what if we simply merged anything we find in the file?